### PR TITLE
Add com.authy.Authy

### PR DESCRIPTION
**Pull request by prodrigestivill**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. *(details below)*
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(none)*

Already contacted with the author in Authy Customer Support with ticket number: 10683058

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by prodrigestivill** [comment]
bot, build com.authy.Authy

**Comment by prodrigestivill** [comment]
bot, build com.authy.Authy

File: com.authy.Authy.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>com.authy.Authy</id>
+    <metadata_license>CC-BY-SA</metadata_license>
```
Comment (hfiguiere) [inline com.authy.Authy.metainfo.xml:4]: this is an invalid metadata license,

also we really encourage to use `CC0-1.0`

Comment (prodrigestivill) [reply com.authy.Authy.metainfo.xml:4]: Thanks for the report, it was an initial example copied from NordPass.
I will fix it in next commit, still want to do some more tests.

**Comment by prodrigestivill** [comment]
bot, build com.authy.Authy

**Comment by barthalion** [comment]
bot, build com.authy.Authy

File: com.authy.Authy.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        sha256: bd26ea4f6b3994b114d52483682fac4baaade38ea248a61480ab9d3b958c3aee
+        size: 109084672
+        x-checker-data:
+          type: snapcraft
+          name: authy
+          channel: stable
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - unsquashfs -dest authy -no-progress -excludes authy.snap meta usr lib command.sh desktop-init.sh desktop-common.sh desktop-gnome-specific.sh
```
Comment (barthalion) [inline com.authy.Authy.yaml:54]: Please silence the standard output.

File: com.authy.Authy.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          - rm -f authy.snap
+      - type: script
+        dest-filename: authy
+        commands:
+          - exec /app/extra/authy/authy --no-sandbox "$@"
```
Comment (barthalion) [inline com.authy.Authy.yaml:59]: Have you checked if it works without `--no-sandbox` when run with [Zypak](https://github.com/refi64/zypak)?

Comment (prodrigestivill) [reply com.authy.Authy.yaml:59]: I tried and with Zypak and XWayland at least, and it seams it don't render the window at all. With wayland I didn't managed to start the application at all with the chromium ozone hint auto options.
Maybe a future release wayland will work.

Also their official snap pakage uses the no sandbox option for now.

**Comment by prodrigestivill** [comment]
@barthalion:
> I don't expect any response from them, but have you reached out to Authy about this submission?

Yes, I edited the main text with a note of it.
I contacted with Authy Customer Support with ticket number 10683058, letting them know about flatpak and flathub. And proposing them to take control of this PR or pronounce regaring this wrapper.

I did this just today, we can wait some days for them to respond...

File: com.authy.Authy.yaml (hunk 2 above)
Comment (prodrigestivill) [reply com.authy.Authy.yaml]: Nice proposal, done in d1dc8aa1454f8a8347f648d8d0217b8565977e4f.

**Comment by prodrigestivill** [comment]
bot, build com.authy.Authy

File: com.authy.Authy.yaml (hunk 3 above)
Comment (barthalion) [reply com.authy.Authy.yaml:59]: Fair, thanks.

**Comment by prodrigestivill** [comment]
Authy replied to my support request 10683058:
> Hi,
>  
> Thanks for reaching out to us. We are very grateful for your feedback and appreciate you taking the time to write to us about your experience with Authy.
>  
> Flatpak is a common petition from our users, and I have added your voice to the existing feature request to help our engineering team prioritize the tasks.
>  
> Unfortunately, I am not able to give you a timeline for these changes to be implemented, but I can assure you that our team is always working on improving our performance and user's experience.
>  
> Let me know if you have additional questions or concerns. I'll be glad to help.
>  
> Best regards,
> _~employee name radacted~_
> Twilio Support

@barthalion for me it is to Flathub to choose what is the best course of actions to be done, either publishing this as it is or waiting a little bit more, both are OK for me.

**Pull request closed without merging**